releasenotes/notes/ddog-gov-urls-0b75b2377e30f77d.yaml

```diff
@@ -0,0 +1,5 @@
+enhancements:
+  - |
+    When using ``site: ddog-gov.com``, the agent now uses agent-version-based
```

Comment:
```suggestion
    When using `site: ddog-gov.com`, the Agent now uses Agent version-based
```

Reply:
This is rST so double-backticks are required.